src/ecosystem/m/querystring/build.rs

```diff
@@ -0,0 +1,104 @@
+use std::borrow::Cow;
```

Comment:
This looks like new functionality?

For now I'd suggest you should put stuff like this in a separate repository/crate. (:

In general I don't want to put too much original stuff in `stdweb`; I just want the very basics here, and anything extra should most likely live as a separate crate. (You might ask then - what is `serde` integration doing here? And the answer to that is simple - `serde` is extremely popular in the Rust community, and you **can't** integrate it with `stdweb` outside of `stdweb` due to Rust's trait implementation coherence rules.)